src/group_theory/group_action.lean

```diff
@@ -193,6 +193,9 @@ class distrib_mul_action (α : Type u) (β : Type v) [monoid α] [add_monoid β]
 (smul_zero {} : ∀(r : α), r • (0 : β) = 0)
 end prio
 
+-- FIXME: this now times out:
+-- example {M : Type*} [add_comm_group M] : has_scalar ℤ M := infer_instance
```

Comment:
Can we understand why this is timing out? How much of a problem is this?

Reply:
I just confirmed that this is fixed in 3.7.0, along with #1561. I'll remove this note now.